Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -54,8 +81,9 @@ static void StartIntro()
             Console.ReadLine();
         }
 
-        static int StartGame()
+        static Dictionary<string, int> StartGame()
         {
+            Dictionary<string, int> playerProgress = new Dictionary<string, int> { };
             int progress = 0;
```

Comment:
this progress variable is not needed anymore because you add result to dictionary so instead you can had local variable inside each `IF` or add `GuessLimit * SingleGuessPoints - guessCount` as second parameter to dictionary `Add` method